wp/wp-content/themes/lahma_maker/page-tags.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/* Template Name: Tags page */
+
+get_header();
+?>
+
+<div id="main" class="site-main" role="main">
+	<div id="content" class="site-content">
+		<div id="primary" class="content-area fullwidth">
+
+            <?php while ( have_posts() ) : the_post(); ?>
```

Comment:
I may be just confused but why / what posts do we show on the tag page? 

Isn't this the tags overview page?